bin/ci-tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+export CI='true'
+set -e
+
+tests=$(ls .test | xargs)
+for test in $tests; do
+  echo "Validating .test/$test values"
+  ln -s $PWD/.test/$test env
+  bats -T bin/tests
+  bin/validate-values.sh
+  bin/validate-templates.sh
+  bin/check-policies.sh
+  unlink env
+done
+
+profiles=$(ls profiles | xargs)
+
+for profile in $profiles; do
```

Comment:
Tests fixtures are already there. I just need to refactor directory names.

BUT I strongly recommend to validate values with profiles as well. Otherwise we may release profiles that does not pass validation.

Reply:
> Tests fixtures are already there. I just need to refactor directory names.
> 
> BUT I strongly recommend to validate values with profiles as well. Otherwise we may release profiles that does not pass validation.

One reason behind splitting up tests and profiles was to not have redundant template testing. I don't want to add more overlap in template testing. If we can't prevent that then we should stick with the profiles as input for tests, and see if we can do 1 + n runs, with n being the number of profiles. So that:

1. we test the common templates
2. we iterate over the profiles and only select the provider dependent templates for the test

To make nr 2 work we have to fix `hf_templates` func as it does not work with named parameters. We can just skip the first part of that function knowing those are not involved when providing named parameters.